Source/WebKit/Scripts/webkit/tests/TestWithEnabledByOrConjunctionMessageReceiver.cpp

```diff
@@ -42,13 +42,15 @@ void TestWithEnabledByOrConjunction::didReceiveMessage(IPC::Connection& connecti
 {
     auto& sharedPreferences = sharedPreferencesForWebProcess();
     UNUSED_VARIABLE(sharedPreferences);
+    bool runtimeEnablementCheckFailed = false;
+    UNUSED_VARIABLE(runtimeEnablementCheckFailed);
     if (!(sharedPreferences.someFeature || sharedPreferences.otherFeature)) {
 #if ENABLE(IPC_TESTING_API)
         if (connection.ignoreInvalidMessageForTesting())
             return;
 #endif // ENABLE(IPC_TESTING_API)
         ASSERT_NOT_REACHED_WITH_MESSAGE("Message %s received by a disabled message receiver TestWithEnabledByOrConjunction", IPC::description(decoder.messageName()).characters());
-        return;
```

Comment:
It seems wrong that return is removed? Wouldn't that make the following code to run even when the feature is disabled?

Reply:
Yes, this is the same issue pointed above.